src/components/gltf-model.js

```diff
@@ -0,0 +1,52 @@
+var registerComponent = require('../core/component').registerComponent;
+var THREE = require('../lib/three');
+
+var GLTFLoader = THREE.glTFLoader; // To appease the linter
+var loader = new GLTFLoader();
+var gltf = null;
+
+module.exports.Component = registerComponent('gltf-model', {
+  schema: {
+    type: 'src'
+  },
+
+  init: function () {
+    this.model = null;
+  },
+
+  update: function () {
+    var self = this;
+    var el = this.el;
+    var src = this.data;
```

Comment:
@ngokevin I'm missing something small here, I think.  this.data points to schema.type['value'], if I change schema to { type: 'src', loop: 'loop', auto: 'auto' } this.data is still just the src value.  I also tried using this.schema directly, but that didn't work and seemed like going around the intended use anyway.